### DISCUSSION TITLE
Please teach GitHub Copilot that vitest exists

### DISCUSSION THREAD

**Discussion by ffxsam** [community/community#62769] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Every time I try to get assistance with unit tests (vitest), it writes code for jest. It seems as if it doesn't know Vitest exists.

**Comment by Akash1134** [comment]
Hi @ffxsam ,

Thanks for sharing and we appreciate your patience. I will share this feedback with the concerned team. You can check our [public roadmap](https://github.com/github/roadmap/projects/1) to see the kinds of work we are focusing on for the coming year. Smaller fixes won’t be listed, but are often included when work is happening adjacent to it.

As a GitHub user, the most you can do is to share your feedback on this Discussion space with the correct tags (include examples), and trust that we are always working to make GitHub as awesome as possible. We can't guarantee that every bit of feedback sees action, but we are paying attention.

Thank You!
